src/shim.js

```diff
@@ -132,7 +161,13 @@ function executeCommand(command) {
         const target = objectSpace[command.target];
         const method = target[command.name];
 
-        if (haveCallbacks.indexOf(command.name) === -1) {
+        if (command.name === 'addEvent') {
+            addEvent(command.target, command.params[0].type, command.params[0].event, command.params[0].args);
+            completeCommand(command);
+        } else if (command.name === 'removeEvent') {
```

Comment:
When I was thinking about how to do it I though the events as different entities than the commands, even if it uses the same bus. That's why I did it that way. If you don't change it, I may fix it tomorrow (and add the docs missing, as updating the Readme) .